internal/commands/server/k8s_e2e_test.go

```diff
@@ -1374,6 +1416,28 @@ func TestRouteParentRefChange(t *testing.T) {
 				listenerAttachedRoutes(1),
 			), checkTimeout, checkInterval, "listeners not ready in the allotted time")
 
+			// Check that HTTPRoute is still routing traffic from first gateway
+			checkRoute(t, firstGatewayCheckPort, "/", httpResponse{
+				StatusCode: http.StatusOK,
+				Body:       serviceOne.Name,
+			}, nil, "service one not routable from first gateway in allotted time")
+
+			// TODO: Routing from multiple gateways is not yet supported.
+			// When implemented, this check should be updated to wait for
+			// http.StatusOK and serviceOne.Name as the body content prefix.
+			checkRoute(t, secondGatewayCheckPort, "/", httpResponse{
+				StatusCode: http.StatusServiceUnavailable,
+				Body:       "no healthy upstream",
+			}, nil, "service one not returning expected error from second gateway in allotted time")
+
+			// Update httpRoute from remote, then remove first gateway ParentRef
```

Comment:
What do you mean when saying `... from remote`?

Reply:
Without a `resources.Get` call to update the local variable here, the subsequent call to `resources.Update` will fail because the version is lower than the version of that object in the cluster (due to gateway controller settings status conditions I believe).

Reply:
Oh so `from remote` here is just pull down an updated copy from the kube API